database/index.js

```diff
@@ -5,8 +5,8 @@ const db = mysql.createConnection(mysqlConfig);
 
 
 const getAllPhotos = (id) => {
-  let queryStr = 'select photo_urls from products where product_id=?;'
-  let params = id;
+  const queryStr = 'select photo_urls from products where product_id=?;';
```

Comment:
I don't believe you need to add the semicolon to the end of querystrings (npm's mysql does that for you in connection.query() I believe)